### PR TITLE
Directory-specific documentation update

### PR DESCRIPTION
**Pull request by e-wai**
Collection of cleanups and updates around README

It's considering the open PRs in `dynamics` and `routemodel`
No update for `batterycelltesting` and `SOC` update is limited (should just go with SOC refactor)

Lots of reading, sorry

### CODE REVIEW

File: routemodel/README.md [hunk 1]
```diff
@@ -1,54 +1,25 @@
...
-2. Methods
-   - *get_elevation_data():* method to request data from API using initialization parameters. No parameters to be entered.
-   - *parse_elevation_data():*  method to parse through API response object. Pass in response from above getter into method call.
-
-## Routes Module
-
-## Tests Module
++ `routes.py`: Methods to query Routes endpoint, collecting information on latitude, longitude, maneuver instructions, distance to maneuver, distance, and street
++ `elevations.py`: Methods to query Elevations endpoint, forming an elevation profile from a set of coordinates
++ `speedlimits.py`: Methods to query Routes endpoint, collecting information around speed limit changes. Separate call from `routes.py` due to specific route points constraints.
```
Comment (e-wai) [inline routemodel/README.md:13]: Two things - 
+ Why are `routes.py` and `speedlimits.py` separate calls?
+ What are the constraints around the above 3 points query builders - can we include this in the docs?

Comment (mdshiozaki) [reply routemodel/README.md:13]: Was it because speedlimits came in from another team so it's not integrated together?

**Review by mdshiozaki — DISMISSED** [review]

File: routemodel/README.md (hunk 1 above)
Comment (ufshk) [reply routemodel/README.md:13]: it's cause they're different endpoints with different constraints

also cause another member originally coded it as a separate file, but i did keep it that way

Comment (e-wai) [reply routemodel/README.md:13]: Can we add the information on the specific API constraints to make it clear then? (I don't know what they are)

**Review by mdshiozaki — APPROVED** [review]

**Pull request merged**